### ISSUE TITLE
Translation Error: The intl string context variable 'nom' was not provided to the string 'Scène activée {nom}.'

### ISSUE THREAD

**Issue by grostim** [home-assistant/home-assistant-polymer#4294]
**Checklist:**

- [ X] I updated to the latest version available
- [ X] I cleared the cache of my browser

**Home Assistant release with the issue:**

Home Assistant 0.102.2

**Last working Home Assistant release (if known):**
not sure. 
**UI (States or Lovelace UI?):**
Version interface utilisateur : 20191119.6 - latest

**Browser and Operating System:**

Windows 10 + Chrome.

**Description of problem:**

It is the first time i am trying to create a scene.
When i try to activate the scene, i have a pop up showing on the left bottom corner:
"Translation Error: The intl string context variable 'nom' was not provided to the string 'Scène activée {nom}.'"

**Javascript errors shown in the web inspector (if applicable):**
None

**Additional information:**

**Comment by bramkragten** [comment]
Placeholders (`{name}` => `{nom}`) should not be translated, you can fix this in Lokalise: https://developers.home-assistant.io/docs/en/internationalization_translation.html

**Comment by grostim** [comment]
Ok, i have corrected it on lokalize. When the modify will be updated on home assistant ? next release ?

**Comment by bramkragten** [comment]
Thanks! Yes, it will be in the next release

**Comment by ronniebee** [comment]
> Placeholders (`{name}` => `{nom}`) should not be translated, you can fix this in Lokalise: https://developers.home-assistant.io/docs/en/internationalization_translation.html

Bram, I'm having the same issue.

Translation Error: The intl string context variable ‘naam’ was not provided to the string 'Geactiveerd {naam].

What do you mean by correct in lokalize? I'm a newbee and just starting with HASSio. I build the script in   "ïnstellingen>scripts".

**Comment by bramkragten** [comment]
Follow these steps https://developers.home-assistant.io/docs/en/internationalization_translation.html and replace `{naam}` with `{name}` in the translation.

**Comment by ronniebee** [comment]
Hi Bram, I have changed (in frontend) naam to name but no changes. I think I don't know wat te do exactly. When I saw naam I replaced it (see example image).

![image](https://user-images.githubusercontent.com/7029899/71780936-00687880-2fc9-11ea-9eaf-4a28f4126b12.png)

The error remains:

Translation Error: The intl string context variable ‘naam’ was not provided to the string 'Geactiveerd {naam].

I must be doing something wrong :-(..
